src/main/jbake/content/blog/2017/07/jaeger-with-security-proxy.adoc

```diff
@@ -0,0 +1,252 @@
+= Protecting Jaeger UI with a sidecar security proxy
+Juraci Paixão Kröhling
+2017-07-18
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger, opentracing
+
+On a production setup of Jaeger, it could be a good idea to restrict access to Jaeger's Query
```

Comment:
Suggested rewording "In a production deployment of Jaeger, it may be advantageous to restrict ..."